clients/java/zms/src/main/java/com/yahoo/athenz/zms/ZMSClient.java

```diff
@@ -1090,8 +1090,8 @@ public Role putRole(String domainName, String roleName, String auditRef, Boolean
      * @param role       role object to be added to the domain
      * @throws ZMSClientException in case of failure
      */
-    public void putRole(String domainName, String roleName, String auditRef, Role role) {
-     putRole(domainName, roleName, auditRef, false, role);
+    public Role putRole(String domainName, String roleName, String auditRef, Role role) {
```

Comment:
we cannot change the existing interface since that will break all of our existing customers. There is no need for this change. We already have a putRole that returns the updated Role object.

same applies to all other changes.